config/initializers/rolify.rb

```diff
@@ -0,0 +1,7 @@
+Rolify.configure do |config|
```

Comment:
Pass `&:use_dynamic_shortcuts` as an argument to `configure` instead of a block.